docs/source/AdministratorGuide/Systems/WorkloadManagement/InputDataResolution.rst

```diff
@@ -0,0 +1,35 @@
+.. _InputDataResolution:
+
+==================================================
+InputDataResolution: giving job access to the data
+==================================================
+
+When a job needs access to data, there are two ways data can be accessed:
+
+* either by downloading the file on the local worker node
+* or by reading the data remotely, aka ``streaming``.
+
+The resolution is done in the ``JobWrapper`` (see :ref:`jobs`). By default, the resolution logic is implemented in :py:mod:`~DIRAC.WorkloadManagementSystem.Client.InputDataResolution`. It can be override by the Job JDL (see `InputDataModule` in :ref:`jdlDescription`), or by the ``/Operations/InputDataPolicy/InputDataModule`` parameter.
```

Comment:
```suggestion
The resolution is done in the ``JobWrapper`` (see :ref:`jobs`). By default, the resolution logic is implemented in :py:mod:`~DIRAC.WorkloadManagementSystem.Client.InputDataResolution`. It can be overridden by the Job JDL (see `InputDataModule` in :ref:`jdlDescription`), or by the ``/Operations/<>/InputDataPolicy/InputDataModule`` parameter.
```